modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -82,14 +84,46 @@ class NewProfileIndex extends React.Component {
       });
     } else {
       let formData = this.state.formData;
+      let guid = '';
+      // Check if GUID fields submitted and generate hash
+      if (
+        formData.firstName
+        && formData.middleName
+        && formData.lastName
+        && formData.placeOfBirth
+      ) {
+        let id = new OpenScienceIdentity({
+          first_name: formData.firstName,
+          middle_name: formData.middleName,
+          last_name: formData.lastName,
+          city_of_birth: formData.placeOfBirth,
```

Comment:
This should always be "city of birth", not "place of birth". The precision is important.

Reply:
hmmm I copied this from openscience, have they been using it wrong ? what are the impacts.

we can take this offline for this one